sys/net/network_layer/ng_sixlowpan/frag/rbuf.c

```diff
@@ -117,7 +105,7 @@ void rbuf_add(ng_netif_hdr_t *netif_hdr, ng_sixlowpan_frag_t *frag,
     }
 
     while (ptr != NULL) {
-        if (_rbuf_int_in(ptr, offset, offset + dg_frag_size - 1)) {
+        if (_rbuf_int_in(ptr, offset, offset + frag_size - 1)) {
             DEBUG("6lo rfrag: overlapping or same intervals, discarding datagram\n");
             ng_pktbuf_release(entry->pkt);
             _rbuf_rem(entry);
```

Comment:
`_rbuf_rem` sets `entry->pkt` to `NULL`. Is this a desired behaviour? What if this `pkt` has several users?